spec/factories/shopify_api/variant_factory.rb

```diff
@@ -22,6 +22,7 @@
     inventory_quantity     { 0 }
     old_inventory_quantity { 0 }
     image_id               { nil }
+    inventory_item_id      { rand(0..1000) }
```

Comment:
do you just need to have here any number?
may be set it to specific number?

Reply:
👍